src/app/components/views/job-view/subviews/job-subview/job-subview.component.html

```diff
@@ -0,0 +1,257 @@
+</div>
+<h6>Open Tickets</h6>
+<div class="font-smaller mb-2">None</div>
+<h6>One Time Charges</h6>
+<div class="font-smaller">None</div>
+<hr>
+<div class="equipment-box">
+  <h2>Equipment</h2>
+  <div class="container">
+    <hr>
+    <div class="row mb-3 mt-4">
+      <div class="col-3">
+        <div class="default-bg-color equipment-icon-box">
+          <img src="assets/fjack.png" alt="fiber jack" width="100%">
+        </div>
+      </div>
+      <div class="col-9">
+        <div class="row ms-1">
+          <div class="col-7 font-smaller">
+            <div class="row">
+              JACK
+            </div>
+            <div class="row">
+              Fiber Jack
+            </div>
+          </div>
+          <div class="col-5 text-end">
+            <div class="btn btn-secondary btn-sm">DETAILS</div>
+          </div>
+        </div>
+        <div class="clean-hr mt-1 mb-1"></div>
+        <div class="row">
+          <div class="col">
+            <span class="dot dot-green me-1"></span>
+            <span class="font-smaller">Installed</span>
+          </div>
+          <div class="col text-end">
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="container">
+    <hr>
+    <div class="row mb-3 mt-4">
+      <div class="col-3">
+        <div class="default-bg-color equipment-icon-box">
+          <img src="assets/bgw320.png" alt="bgw320" width="100%">
+        </div>
+      </div>
+      <div class="col-9">
+        <div class="row ms-1">
+          <div class="col-7 font-smaller">
+            <div class="row">
+              ONT
+            </div>
+            <div class="row">
+              IONT320-505-X
+            </div>
+          </div>
+          <div class="col-5 text-end">
+            <div class="btn btn-secondary btn-sm">DETAILS</div>
+          </div>
+        </div>
+        <div class="clean-hr mt-1 mb-1"></div>
+        <div class="row">
+          <div class="col">
+            <span class="dot dot-green me-1"></span>
+            <span class="font-smaller">Connected</span>
+          </div>
+          <div class="col text-end font-smaller">
+            <span>SIG:-19.2 dBm</span>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="container">
+    <hr>
+    <div class="row mb-3 mt-4">
+      <div class="col-3">
+        <div class="default-bg-color equipment-icon-box">
+          <img src="assets/bgw320.png" alt="bgw320" width="100%">
+        </div>
+      </div>
+      <div class="col-9">
+        <div class="row ms-1">
+          <div class="col-7 font-smaller">
+            <div class="row">
+              ROUTER/GATEWAY
+            </div>
+            <div class="row">
+              Nokia NGW320-505
+            </div>
+          </div>
+          <div class="col-5 text-end">
+            <div class="btn btn-secondary btn-sm">DETAILS</div>
+          </div>
+        </div>
+        <div class="clean-hr mt-1 mb-1"></div>
```

Comment:
you could put `my-1` here instead of mt and mb

Reply:
done